exercises/pov/description.md

```diff
@@ -5,25 +5,27 @@ point of view. For example family trees are usually presented from the
 ancestor's perspective:
 
 ```
-          0
-          |
-    1-----2-----3
-    |     |     |
-  4-+-5 6-+-7 8-+-9
+    +------0------+
+    |      |      |
+  +-1-+  +-2-+  +-3-+
+  |   |  |   |  |   |
+  4   5  6   7  8   9
 ```
 
 But the same information can be presented from the perspective of any other node
 in the graph, by pulling it up to the root and dragging its relationships along
 with it. So the same graph from 6's perspective would look like:
 
 ```
-        6
-        |
-  7-----+-----2
+  +-----6-----+
+  |           |
+  7           2
```

Comment:
Per the *ancestor's perspective* graph above, shouldn't there be an edge between `7` and `2` here?